linux/xdg-desktop-portal/build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+plugins {
+    java
+    id("dev.nokee.jni-library")
+    id("dev.nokee.cpp-language")
+    `uber-jni-jar`
+    `use-prebuilt-binaries`
```

Comment:
If you don't plan to interface with any C/C++ code you don't have to add these plugins

Reply:
Makes sense, I deleted those parts